src/libos/src/unit_test/test.rs

```diff
@@ -0,0 +1,7 @@
+use super::*;
+
+#[occlum_test]
+#[should_panic]
+pub fn test_should_panic() {
+    panic!();
+}
```

Comment:
This is too trivial to be a separate file.

Can we have a meaningful `should_panic` unit test somewhere? Or this example can be added as a code snippet to the module-level Rust doc of `unit_test` module.

Reply:
Basically, this is a test for `unit_test`. I put it close to `unit_test`.

Reply:
Ok